report_templates/host_-_all_installed_packages.erb

```diff
@@ -0,0 +1,24 @@
+<%#
+name: Host - All Installed Packages
+snippet: false
+template_inputs:
+- name: hosts
+  required: false
+  input_type: user
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
```

Comment:
I'm pretty sure this shouldn't be here

Reply:
> I'm pretty sure this shouldn't be here

I just create a new report template and export it, that said, this is part of the export process. I'm not sure if is totally mandatory during the installation/creation. If not, let me know and I'll update the commit.

Thank you

Reply:
the resource_type is listed even though only useful for value_type: search

I actually think having that instead of plain would be beneficial, this makes the field smarted and allows autocompletion in the generate form, @waldirio could you please update that?

Reply:
hey @ares change made. Thank you!